docs/experimentalists/pooler/random/index.md

```diff
@@ -22,8 +22,9 @@ This means that there are 9 possible combinations for these variables (3x3), fro
 | 3  | (3,4) | (3,5) | X   |
 
 ### Example Code
+
 ```python
-from autora.experimentalist.pooler.random_pooler import random_pool
+from autora.experimentalist.random_ import random_pool
 
-pool = random_pool([1, 2, 3],[4, 5, 6], n=3)
+pool = random_pool([1, 2, 3], [4, 5, 6], n=3)
```

Comment:
should it be `num_samples` here instead of `n`?

on a related note, this prompted me to look through the `random_.py` file; why is it not also `num_samples=num_samples` in [line 339](https://github.com/AutoResearch/autora-core/blob/221302d5a4df65ba7c20a7b24fd87e514bf871fe/src/autora/experimentalist/random_.py#L339)

Reply:
Yes, great catch, Ben! It should be `num_sampled`; not `n`.

Reply:
Yes!